internal/org/model/types.go

```diff
@@ -0,0 +1,20 @@
+package model
+
+import "github.com/caos/zitadel/internal/eventstore/models"
+
+const (
+	OrgAggregate models.AggregateType = "org"
+
+	OrgAdded       models.EventType = "org.added"
+	OrgChanged     models.EventType = "org.changed"
+	OrgDeactivated models.EventType = "org.deactivated"
+	OrgReactivated models.EventType = "org.reactivated"
+
+	OrgMemberAdded   models.EventType = "org.member.added"
+	OrgMemberChanged models.EventType = "org.member.changed"
+	OrgMemberRemoved models.EventType = "org.member.removed"
+
+	GrantMemberAdded   models.EventType = "org.grant.member.added"
```

Comment:
I would name it orggrantmember, because we have also other grants, if this types will be used in mgmt api you dont know which grant it is

Reply:
deleted because wrong PR